pallets/parachain-staking/src/lib.rs

```diff
@@ -2468,11 +2522,13 @@ pub mod pallet {
 		/// Weight: O(D) where D is the number of delegators of this collator
 		/// block author bounded by `MaxDelegatorsPerCollator`.
 		/// - Reads: CollatorState, Total, Balance, InflationConfig,
-		///   MaxSelectedCandidates
-		/// - Writes: D * Balance
+		///   MaxSelectedCandidates, Validators, DisabledValidators
+		/// - Writes: (D + 1) * Balance
 		/// # </weight>
 		fn note_author(author: T::AccountId) {
-			// should always include state
+			log::info!("current author {:?}", pallet_authorship::Pallet::<T>::author());
```

Comment:
we log on each block? Not sure if we want to have that as a `debug`. 🤔 Normally we log very few things...

Reply:
Good catch, this was used to debug but the debug logs spammed my terminal too much.